metadata/src/audio/item.rs

```diff
@@ -16,98 +17,268 @@ use librespot_core::{
 
 pub type AudioItemResult = Result<AudioItem, Error>;
 
-// A wrapper with fields the player needs
 #[derive(Debug, Clone)]
 pub struct AudioItem {
-    pub id: SpotifyId,
-    pub spotify_uri: String,
+    pub track_id: SpotifyId,
+    pub uri: String,
```

Comment:
My thought was that uri is an MPRIS metadata field so eventually you'd have to unwrap it anyway. But it's completely up to you.

Reply:
Yeah, that's right. I guess, it's minor enough that doing it in the MPRIS code doesn't hurt, and we can go for a slightly smaller struct size here.

Reply:
No problem I'll change it when I get home along with your Covers struct.

Reply:
On 2nd thought uri is also used at least once in player it might stay?

Reply:
Hmm, you're probably right. Since we're doing that allocation anyway, we can also leave it in the struct… No strong opinion on this, I'll leave that up to you.